### PR TITLE
Add space between rows in IDE list

### PR DESCRIPTION
**Pull request by aymyers**
Add padding-bottom of 15px to IDE list items so they aren't so close together on https://www.dartlang.org/tools

In response to issue #620

### CODE REVIEW

File: src/_assets/stylesheets/main.scss [hunk 1]
```diff
@@ -710,6 +710,7 @@ main .content {
   column-gap: 0;
```
Comment (chalin) [inline src/_assets/stylesheets/main.scss]: For `.col2` I'd suggest adding:
```css
  padding-left: $font-size-base / 2;
```

File: src/_assets/stylesheets/main.scss [hunk 2]
```diff
@@ -710,6 +710,7 @@ main .content {
...
   li {
     page-break-inside: avoid;
+    padding-bottom: 15px;
```
Comment (chalin) [inline src/_assets/stylesheets/main.scss]: I'd prefer
```css
    padding: $font-size-base / 2;
    font-size: smaller;
```

**Comment by chalin** [comment]
The extra CSS style changes above give make this look like this on mobile:

> screen shot 2018-03-04 at 15 50 23


A few other things we can do to clean this up (@aymyers, you don't have to do these unless you'd like the extra challenge :):

- Add `class="no-automatic-external"` to the anchors; that view is busy enough without the external link icon.
- Convert the `<ul>` HTML list markup to markdown. To add a class to the list, put `{:.col2}` just before the first markdown list entry.

@kwalrath - WDYT about this extra suggestions. @aymyers - feel free to ignore these suggestions (certainly at least until @kwalrath chims in).

**Comment by chalin** [comment]
@kwalrath - I'll merge now. We can make any further adjustments in a followup PR.

**Comment by kwalrath** [comment]
Thanks very much for the contribution, @aymyers!

**Comment by aymyers** [comment]
You're welcome!

**Pull request merged**